content/docs/pulumi-cloud/access-management/_index.md

```diff
@@ -13,6 +13,8 @@ menu:
     pulumicloud:
         weight: 7
         identifier: access-management
+aliases:
+  - /docs/pulumi-cloud/access-management/
```

Comment:
I don't think this alias will do anything, since it's the same path as the page. Right?